kvdb/kvdb_no_sqlite.go

```diff
@@ -0,0 +1,21 @@
+//go:build !kvdb_sqlite || (windows && (arm || 386)) || (linux && (ppc64 || mips || mipsle || mips64))
```

Comment:
We can use these constants to print out the OS/arch combo not supported: https://pkg.go.dev/runtime#pkg-constants

> So SQLite won't be available on Windows i386/ARM and any PPC/MIPS platform?

So windows amd64+arm64 are supported (no 32 bit), a nd the same for ppc (no mips at all): https://pkg.go.dev/modernc.org/sqlite#hdr-Supported_platforms_and_architectures.